Doc/library/pathlib.rst

```diff
@@ -1302,6 +1302,7 @@ Below is a table mapping various :mod:`os` functions to their corresponding
 :func:`os.path.samefile`               :meth:`Path.samefile`
 :func:`os.path.splitext`               :data:`PurePath.stem` and
                                        :data:`PurePath.suffix`
+:func:`os.walk`                        :meth:`Path.glob` and :meth:`Path.rglob`
```

Comment:
These aren't similar enough to appear in this table without a lengthy footnote below. Also see #90385

Reply:
Actually more importantly is that I'm expecting https://github.com/python/cpython/pull/92517 to eventually land which will directly replace `os.walk()`.

```suggestion
```